### PR TITLE
[17.0][MIG] stock_request_submit: Migration to 17.0

### PR DESCRIPTION
**Pull request by peluko00**
Module migrated to version 17.0

cc https://github.com/APSL 155222

@miquelalzanillas @lbarry-apsl @javierobcn @mpascuall please review

- [Issue] https://github.com/OCA/stock-logistics-request/issues/10

### CODE REVIEW

**Review by mpascuall — APPROVED** [review]
Working good, checked in runboat

**Review by lbarry-apsl — APPROVED** [review]
LGTM, revised functionality

**Comment by peluko00** [comment]
Ready for merge @rousseldenis ?

File: stock_request_submit/models/stock_request_order.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+class StockRequestOrder(models.Model):
+    _inherit = "stock.request.order"
+
+    def action_submit(self):
+        if not self.stock_request_ids:
+            raise UserError(
+                _("There should be at least one request item for submiting the order.")
+            )
+        for line in self.stock_request_ids:
+            line.action_submit()
```
Comment (rousseldenis) [inline stock_request_submit/models/stock_request_order.py]:
```suggestion
            self.stock_request_ids.action_submit()
```

?

Comment (peluko00) [reply stock_request_submit/models/stock_request_order.py]: Done!

**Comment by peluko00** [comment]
I did a refactor on tests, now it's ready

**Review by max3903 — APPROVED** [review]

**Comment by peluko00** [comment]
It's ready @max3903

**Pull request merged**